client/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
```

Comment:
clean the readme to bare min setup instructions and add a screenshot of what to expect.